skimage/io/tests/test_pil.py

```diff
@@ -97,6 +97,17 @@ def test_imread_truncated_jpg():
                   os.path.join(data_dir, 'truncated.jpg'))
 
 
+def test_jpg_quality_arg():
+    chessboard = np.load(os.path.join(data_dir, 'chessboard_GRAY_U8.npy'))
+    with NamedTemporaryFile(suffix='.jpg') as jpg:
```

Comment:
Based on the [docs](https://docs.python.org/2/library/tempfile.html#tempfile.NamedTemporaryFile), I think you'll need to close the temporary file before calling `imsave` on Windows.